src/model/mapping/group_views_model.py

```
@@ -0,0 +1,97 @@
+
+# Init tqdm for pandas
+tqdm.pandas()
+
+"""
+Виды номенклатур определяются по внутренней группе (internal_group),
+а не по НСИшной (group)
+"""
+
+# GROUP_VIEWS_TABLE_PATH = f"{DATA_FOLDER_PATH}/levelgroup-group-views.xlsx"
+# ! REMOVE
+GROUP_VIEWS_TABLE_PATH = f"/home/syrnnik/Documents/Syrnnik/BandaWorks/Datastep/demo-backend/data/levelgroup-group-views.xlsx"
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)